'transformers.models.janus.convert_janus_weights_to_hf' is missing from release

### System Info

In transformers v.4.52.0 - v4.52.2, the file: src/transformers/models/janus/convert_janus_weights_to_hf.py from #36053 is missing and results in an `ModuleNotFoundError` when running tests/models/janus/test_processor_janus.py:
```
Traceback:
tests/models/janus/test_processor_janus.py:23: in <module>
    from transformers.models.janus.convert_janus_weights_to_hf import CHAT_TEMPLATE
E   ModuleNotFoundError: No module named 'transformers.models.janus.convert_janus_weights_to_hf'
```

cc: @ArthurZucker @zucchini-nlp @yaswanth19

### Who can help?

_No response_

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

1. Run test `tests/models/janus/test_processor_janus.py`
2. Test fails with ModuleNotFoundError.

### Expected behavior

Test passes.

--- a/tests/models/janus/test_processor_janus.py
+++ b/tests/models/janus/test_processor_janus.py
@@ -44,6 +44,9 @@ class JanusProcessorTest(ProcessorTesterMixin, unittest.TestCase):
             "deepseek-community/Janus-Pro-1B",
             extra_special_tokens=special_image_tokens,
         )
+        # Set the processor to use the default system prompt to False as it's used based on input modality.
+        # Hence set to False to avoid any issues in the test irrespective of inputs.
+        processor.use_default_system_prompt = False
         processor.save_pretrained(self.tmpdirname)
 
     def get_tokenizer(self, **kwargs):
@@ -53,11 +56,7 @@ class JanusProcessorTest(ProcessorTesterMixin, unittest.TestCase):
         return AutoProcessor.from_pretrained(self.tmpdirname, **kwargs).image_processor
 
     def get_processor(self):
-        processor = AutoProcessor.from_pretrained(self.tmpdirname)
-        # Set the processor to use the default system prompt to False as it's used based on input modality.
-        # Hence set to False to avoid any issues in the test irrespective of inputs.
-        processor.use_default_system_prompt = False
-        return processor
+        return AutoProcessor.from_pretrained(self.tmpdirname)
 
     def test_chat_template_single(self):
         """

--- a/tests/models/janus/test_processor_janus.py
+++ b/tests/models/janus/test_processor_janus.py
@@ -20,7 +20,6 @@ import unittest
 import numpy as np
 
 from transformers import AutoProcessor, AutoTokenizer, JanusProcessor
-from transformers.models.janus.convert_janus_weights_to_hf import CHAT_TEMPLATE
 from transformers.utils import is_vision_available
 
 from ...test_processing_common import ProcessorTesterMixin
@@ -53,11 +52,12 @@ class JanusProcessorTest(ProcessorTesterMixin, unittest.TestCase):
     def get_image_processor(self, **kwargs):
         return AutoProcessor.from_pretrained(self.tmpdirname, **kwargs).image_processor
 
-    def prepare_processor_dict(self):
-        # similar to Emu3 and Qwen2VLProcessorTest, but keep the template in the convert script to avoid duplicated code
-        return {
-            "chat_template": CHAT_TEMPLATE,
-        }
+    def get_processor(self):
+        processor = AutoProcessor.from_pretrained(self.tmpdirname)
+        # Set the processor to use the default system prompt to False as it's used based on input modality.
+        # Hence set to False to avoid any issues in the test irrespective of inputs.
+        processor.use_default_system_prompt = False
+        return processor
 
     def test_chat_template_single(self):
         """

